package.json

```diff
@@ -1719,12 +1719,6 @@
             "default": null,
             "description": "%configuration.dotnet.server.crashDumpPath%"
           },
-          "dotnet.enableXamlToolsPreview": {
```

Comment:
Lets keep the option, but change the default

Reply:
The default is already `true`. The idea (and plan) is to remove the option altogether for release.